src/LoginContainer/LoginContainer.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './LoginContainer.css';
+// import background from '../assets/background.png';
+import LoginForm from '../LoginForm/LoginForm'
+
+class LoginContainer extends Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <LoginForm />
+    )
+  }
+}
+
+export default LoginContainer;
```

Comment:
We may be able to make this a functional component eventually because all it does is render. Let's talk about it and decide for sure later. It's fine like this for now. Love how cleanly this is organized so far!